### PR TITLE
Fix/export bar in it

### PR DESCRIPTION
**Pull request by rbioteau**
Update IT tests to support new bar generation constraints

### CODE REVIEW

File: tests/org.bonitasoft.studio.exporter.tests/src/org/bonitasoft/studio/exporter/tests/autologin/TestAutoLogin.java [hunk 1]
```diff
@@ -1,54 +1,57 @@
 /**
  * Copyright (C) 2013 BonitaSoft S.A.
```
Comment (apupier) [inline tests/org.bonitasoft.studio.exporter.tests/src/org/bonitasoft/studio/exporter/tests/autologin/TestAutoLogin.java]: 2013-2015 and BonitaSoft --> Bonitasoft (je sais c'est relou)

File: bundles/plugins/org.bonitasoft.studio.engine/src/org/bonitasoft/studio/engine/operation/DeployProcessOperation.java [hunk 2]
```diff
@@ -386,4 +382,12 @@ protected ProcessEnablementProblemsDialog createProcessEnablementProblemsDialog(
                 process, processResolutionProblems);
     }
 
+    /**
+     * @return
+     */
+    public IStatus getStatus() {
+        // TODO Auto-generated method stub
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.engine/src/org/bonitasoft/studio/engine/operation/DeployProcessOperation.java]: //TODO

File: bundles/plugins/org.bonitasoft.studio.engine/src/org/bonitasoft/studio/engine/operation/DeployProcessOperation.java [hunk 3]
```diff
@@ -386,4 +382,12 @@ protected ProcessEnablementProblemsDialog createProcessEnablementProblemsDialog(
...
+        return null;
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.engine/src/org/bonitasoft/studio/engine/operation/DeployProcessOperation.java]: avoid returning a null status?

File: tests/org.bonitasoft.studio.tests/src/org/bonitasoft/studio/tests/parameter/TestProcessParameters.java [hunk 4]
```diff
@@ -44,7 +45,6 @@
 
 /**
  * @author Romain Bioteau
- *
  */
 public class TestProcessParameters extends TestCase {
```
Comment (apupier) [inline tests/org.bonitasoft.studio.tests/src/org/bonitasoft/studio/tests/parameter/TestProcessParameters.java]: migrate this one also to JUnit4? (not blocking for merge)

Comment (rbioteau) [inline tests/org.bonitasoft.studio.tests/src/org/bonitasoft/studio/tests/parameter/TestProcessParameters.java]: Done

**Pull request merged**